pkg/flare/diagnose.go

```diff
@@ -0,0 +1,20 @@
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+package flare
+
+import (
+	"io"
+
+	"github.com/DataDog/datadog-agent/pkg/diagnose"
+)
+
+var diagnoseNames = []string{
+	"Kubernetes API Server availability",
```

Comment:
you created a `const` but did not reuse it here

Reply:
I didn't used it to avoid adding a build tag for kubeapiserver in flare, I think it's more important to keep them independent than to import the const, right?